pulsar/impl_message_test.go

```diff
@@ -36,6 +35,11 @@ func TestMessageId(t *testing.T) {
 	assert.Equal(t, int32(3), id2.(messageID).batchIdx)
 	assert.Equal(t, int32(4), id2.(messageID).partitionIdx)
 
+	assert.Equal(t, int64(1), id2.LedgerId())
+	assert.Equal(t, int64(2), id2.EntryId())
+	assert.Equal(t, int32(3), id2.BatchIdx())
+	assert.Equal(t, int32(4), id2.PartitionIdx())
+
```

Comment:
Please do not cover the current function in the original test case. Can you add a new test case to cover this feature?

Reply:
ok.